Core/Core/Helpers/BinaryFormatter.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.IO;
+using System.Text;
+
+namespace Speckle.Core.Helpers;
+
+/// <summary>
+/// Mimics the behaviour of System.Runtime.Serialization.Formatters.Binary.BinaryFormatter for compatibility with Dotnet 8
+/// </summary>
+public static class BinaryFormatter
+{
+  /// <summary>
+  /// Leading bytes on a BinaryFormatter (should be consistent, though I haven't found an actual definition)
+  /// </summary>
+  private static readonly byte[] s_prefix = new byte[] { 0, 1, 0, 0, 0, 255, 255, 255, 255, 1, 0, 0, 0, 0, 0, 0, 0, 6, 1, 0, 0, 0 };
```

Comment:
where does this magic array of numbers come from? be good to elucidate what they are.